docs/source/test_network.md

```diff
@@ -0,0 +1,586 @@
+to be signed by Org1 and Org2, any chaincode invoke using the peer binaries
+needs to target both `peer0.org1.example.com` and `peer0.org1.example.com`. You
+can pass the following values to your invoke commands to target both peers:
+```
+ --peerAddresses localhost:7051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt --peerAddresses localhost:9051 --tlsRootCertFiles ${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+```
+
+The section above passes the endpoints of both peers to the `--peerAddresses`
+flag, and provides the path to each peer's TLS certificate in the organizations
+folder to the `--tlsRootCertFiles` flags.
+
+After we invoke the chaincode, we can use another query to see how the invoke
+changed the assets on the blockchain ledger. Since we already queried the Org1
+peer, we can take this opportunity to query the chaincode running on the Org2
+peer. Set the following environment variables to operate as Org2:
+```
+# Environment variables for Org2
+
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_LOCALMSPID="Org2MSP"
+export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
+export CORE_PEER_ADDRESS=localhost:9051
+```
+
+You can now query the fabcar chaincode running on `peer0.org2.example.com`:
+```
+peer chaincode query -C mychannel -n fabcar -c '{"Args":["queryAllCars"]}'
+```
+
+The result will show that `"CAR9"` was transferred to Dave:
+```
+[{"Key":"CAR0", "Record":{"make":"Toyota","model":"Prius","colour":"blue","owner":"Tomoko"}},{"Key":"CAR1", "Record":{"make":"Ford","model":"Mustang","colour":"red","owner":"Brad"}},{"Key":"CAR2", "Record":{"make":"Hyundai","model":"Tucson","colour":"green","owner":"Jin Soo"}},{"Key":"CAR3", "Record":{"make":"Volkswagen","model":"Passat","colour":"yellow","owner":"Max"}},{"Key":"CAR4", "Record":{"make":"Tesla","model":"S","colour":"black","owner":"Adriana"}},{"Key":"CAR5", "Record":{"make":"Peugeot","model":"205","colour":"purple","owner":"Michel"}},{"Key":"CAR6", "Record":{"make":"Chery","model":"S22L","colour":"white","owner":"Aarav"}},{"Key":"CAR7", "Record":{"make":"Fiat","model":"Punto","colour":"violet","owner":"Pari"}},{"Key":"CAR8", "Record":{"make":"Tata","model":"Nano","colour":"indigo","owner":"Valeria"}},{"Key":"CAR9", "Record":{"make":"Holden","model":"Barina","colour":"brown","owner":"Dave"}}]
+```
+
+## Bring down the network
+
+When you are finished using the test network, you can bring down the network
+with the following command:
+```
+./network.sh down
+```
+
+The command will stop and remove the node containers, remove the crypto material
+and the system channel genesis block, and delete the chaincode images from your
+Docker Registry.
```

Comment:
Does your `./network.sh up` command create docker volumes, like `./byfn.sh up`? If so does `./network.sh down` also remove the docker volumes. Historically that is the #1 problem community members report is that they run `./byfn.sh up` and it fails, they then try to run `./byfn.sh up` again and they get even worse results. It's not obvious they need to run `./byfn.sh down` to delete the volumes in between runs. If it does delete the docker volumes, I think its worth highlighting, even bolding that this happens, and is required to restart the network.

Reply:
Yes and yes.

Reply:
Added the following sentence to this section: `The command also removes the channel artifacts and docker volumes from previous runs, allowing you to run `./network.sh up` again if you encountered any problems.`

Also added volumes to the first point in the troubleshooting section.